docs/01-overview/main-areas/eventing/evnt-01-overview.md

```diff
@@ -2,6 +2,35 @@
 title: Overview
 ---
 
-Eventing in Kyma is used to send and receive events from applications. For example, you can subscribe to events from an external application, and when the user performs an action there, you can trigger your Function or microservice. To subscribe to events, you need to use the Kyma [Subscription CR](../../../05-technical-reference/00-custom-resources/evnt-01-subscription.md).
+Eventing, in general, consists of event producers (or publishers) and consumers (or subscribers) that send events to or receive events from an event processing backend. The PubSub messaging pattern, which is the eventing paradigm used in Kyma, does not send messages directly to the subscribers but instead sends them to particular topics (or event types) without any knowledge of existing subscribers. The diagram shows a high level overview of PubSub communication:
 
-Kyma supports [Cloud Events](https://cloudevents.io/) - a common specification for describing event data - and legacy events. Legacy events are converted to Cloud Events by Kyma.
+![PubSub](./assets/pubsub.svg)
+
+The key benefits of PubSub are:
+- communication between loosely coupled components.
+- scale publishers and subscribers easily.
+- asynchronous communication.
+
+## Eventing in Kyma in a nutshell
+
+The objective of Eventing in Kyma is to simplify the process of publishing and subscribing to events. Kyma uses proven Eventing backend technology such as [NATS JetStream](https://docs.nats.io/), so that the user can achieve a seamless experience with their end-to-end business flow. The user does not have to implement or integrate any intermediate backend or protocol. Eventing in Kyma from a user's perspective works as follows:
+
+ - Offer an HTTP end point, for example a lambda function to receive the events.
+ - Specify the events the user is interested in using the Kyma [Subscription CR](../../../05-technical-reference/00-custom-resources/evnt-01-subscription.md).
+ - Send Legacy or [Cloud Events](https://cloudevents.io/) to configured HTTP end points on our [Event Publishing Proxy](https://github.com/kyma-project/kyma/tree/main/components/event-publisher-proxy) service.
+   - `/publish` for Cloud Events.
+   - `<application_name>/v1/events` for Legacy events.
+ 
+Read [Eventing architecture](../../../05-technical-reference/00-architecture/evnt-01-architecture.md) to know more about the eventing flow in Kyma.
+
+## Glossary for Eventing
```

Comment:
That sounds like a good idea. But I think it doesn't fit in with those items since they are high level components or K8s related terms. Can we have an eventing subsection?

Reply:
Added them as a new subsection, as discussed!